main.nf

```diff
@@ -1,13 +1,34 @@
-
 // Find your tower s3 bucket and upload your input files into it
 // The tower space is PHI safe
 nextflow.enable.dsl = 2
 
+// Synapse ID for Submission View
 params.view_id = "syn51356905"
-params.input_dir = "${projectDir}/input"
+// Synapse ID for Input Data folder
+params.input_id = "syn51390589"
```

Comment:
Nit: Might be worthwhile to note that these synapse ids won't work for non DPE members

Reply:
added in the latest commit